Mathlib/Topology/ContinuousMap/Bounded/Basic.lean

```diff
@@ -626,7 +626,7 @@ instance instLipschitzAdd : LipschitzAdd (α →ᵇ β) where
       rw [dist_le (mul_nonneg C_nonneg dist_nonneg)]
       intro x
       refine le_trans (lipschitz_with_lipschitz_const_add ⟨f₁ x, g₁ x⟩ ⟨f₂ x, g₂ x⟩) ?_
-      refine mul_le_mul_of_nonneg_left ?_ C_nonneg
+      gcongr
       apply max_le_max <;> exact dist_coe_le_dist x⟩
```

Comment:
Should `max_le_max` be `gcongr` too?

Reply:
`sup_le_sup` is a `gcongr` lemma, but here the goal is about `dist` in `Prod _ _`, so applying `max_le_max` unfolds some definitions.